gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -204,7 +209,7 @@ def _annotate_vcf_files(input_pattern, known_args, pipeline_args):
 
   with beam.Pipeline(options=options) as p:
     _ = (p
-         | beam.Create([input_pattern])
+         | beam.Create(input_patterns)
```

Comment:
I think it would be useful to add/modify integration test for annotation, which runs with multiple input_patterns. You can modify the case `gnomad_genomes_GRCh37_chrX_head2500_run_vep` (by updating the input_pattern to input_file).

Reply:
Done.